examples/iterators/emitter.php

```diff
@@ -26,23 +23,15 @@
             $emitter->emit(8);
             $emitter->emit(9);
             $emitter->emit(10);
-            $emitter->resolve(11);
+            $emitter->complete();
         });
 
-        $stream = $emitter->stream();
+        $iterator = $emitter->getIterator();
 
-        $generator = function (Stream $stream) {
-            $listener = new StreamIterator($stream);
-
-            while (yield $listener->advance()) {
-                printf("Stream emitted %d\n", $listener->getCurrent());
-                yield new Pause(100); // Listener consumption takes 100 ms.
-            }
-
-            printf("Stream result %d\n", $listener->getResult());
-        };
-
-        yield new Coroutine($generator($stream));
```

Comment:
No idea honestly… probably some leftover from code being refactored from the async-interop loop. I think it was wrapped with `wait()` at one point.